docs/how-to/buildPingDirectoryProfile.md

```diff
@@ -0,0 +1,169 @@
+  provided by manage-profile
+
+  Some changes may need to be made to the generated profile. Any desired LDIF
+  files will need to be added to the profile. Any additional server root files,
+  server SDK extensions, and dsconfig commands can be manually added, and
+  variables-ignore.txt can be updated to ignore certain files during variable
+  substitution. See the README file at /tmp/pd.profile/misc-files/README for
+  more information on the manual steps that must be taken for the generated
+  profile to be used with the manage-profile tool
+
+  The following files and directories in the server root were excluded from the
+  generated profile, and can be manually added if necessary. These files can
+  also be included by generate-profile with the --includePath argument:
+  config/truststore
+  config/ads-truststore
+  config/encryption-settings.pin
+  config/tools.properties.orig
+  config/encryption-settings/encryption-settings-db
+  config/keystore.p12
+  config/tools.properties
+  config/encryption-settings/encryption-settings-db.old
+  config/keystore.pin
+  config/keystore
+  config/ads-truststore.pin
+  config/truststore.p12
+  config/truststore.pin
+
+  The generated profile can be found at /tmp/pd.profile
+  ```
+
+!!! Info "Note other paths that are not included"
+    The `manage-profile generate-profie` command outputs valuable information about what is and isn't included in the generated profile. 
+
+!!! warning "Don't put secrets in your profile!"
+    Secrets should not be included in your profile, so they are not included in the profile generation by default. 
+    However, If you have not already added encryption secrets or keystores to your environment, you can use the `--includePath` argument to collect items from the running server. These items should then be provided to the server on it's next restart via some secrets management tool. 
+
+
+### Extract Generated Profile
+
+Following the kubernetes example, you can copy out the generated profile with: 
+
+  ```
+  kubectl cp pingdirectory-0:/tmp/pd.profile pd.profile
+  ```
+
+  Sample output:
+  ```
+  % tree
+  .
+  └── pd.profile
+      ├── dsconfig
+      │   └── 00-config.dsconfig
+      ├── ldif
+      │   └── userRoot
+      ├── misc-files
+      │   └── README
+      ├── server-root
+      │   ├── post-setup
+      │   └── pre-setup
+      │       ├── PingDirectory.lic
+      │       ├── README.md
+      │       └── config
+      │           ├── encryption-settings.pin ## Added via --includePath
+      │           ├── keystore.pin ## Added via --includePath
+      │           └── schema
+      │               ├── 80-format-counter-metrics.ldif
+      │               ├── 87-local-identities.ldif
+      │               ├── 88-grants.ldif
+      │               ├── 89-sessions.ldif
+      │               └── 90-oauth-clients.ldif
+      ├── server-sdk-extensions
+      ├── setup-arguments.txt
```

Comment:
We should recommend removing setup-arguments.txt, since the more typical flow in our images is for it to be generated

Reply:
oh yea! missed that step
Thanks!